pkg/azure/actuator.go

```diff
@@ -257,13 +204,18 @@ func (a *Actuator) Delete(ctx context.Context, cr *minterv1.CredentialsRequest)
 		string(credentialsRootSecret.Data[AzureSubscriptionID]),
 	)
 	if err != nil {
+		// TODO: Is it okay to treat this as an error worthy of retrying?
 		return fmt.Errorf("unable to create azure cred minter: %v", err)
```

Comment:
Now with the passthrough steady-state, there really should never be any Azure API calls being made at all. So any kind of failure to make an Azure client is basically not a big deal.
But I agree, I don't really see why this should/would fail as there are no creds involved at that point.